spec/js_routes/options_spec.rb

```diff
@@ -466,7 +466,7 @@
 
     context "with url_links option" do
       around(:each) do |example|
-        ActiveSupport::Deprecation.silence do
+        (Rails.try(:deprecator) || ActiveSupport::Deprecation).silence do
```

Comment:
Same here, prefer `Rails.version` check if possible.

Reply:
Idea about such fix was taken from this - https://github.com/rails-api/active_model_serializers/pull/2483/files